Assets/Scripts/PlayerScripts/PlayerAction.cs

```diff
@@ -44,18 +46,29 @@ public abstract class PlayerAction : MonoBehaviour
 
     protected int GetAccuracy(double timeStamp, int inputIndex)
     {
-        if (Math.Abs(AudioTime - (timeStamp)) < MarginOfError)
+        if (Math.Abs(AudioTime - (timeStamp)) < PerfectMarginOfError)
         {
+            //Perfect
             Hit();
             print($"Hit on {inputIndex} note - time: {timeStamp} audio time {AudioTime}");
             inputIndex++;
         }
-        else
+        else if (Math.Abs(AudioTime - (timeStamp)) < NiceMarginOfError)
         {
+            //Nice
             Inaccurate();
```

Comment:
Since this else-if block for "Nice!" doesn't increase inputIndex, you can spam the jump button in-between 0.6 and 0.3 seconds and get a LOT of points

Reply:
True, I'll add that

Reply:
Also because inputIndex isn't updated when you get Nice!, you always get an Oops! right after